src/views/Player.vue

```diff
@@ -376,13 +396,28 @@ export default {
 
     async setPlayerProperties(player) {
       player.on("ready", () => {
+        // start playing from 5 seconds before where the user left off in previous session
+        if (this.previousPlayerTime > stepBackTime){
+          this.player.currentTime = this.previousPlayerTime - stepBackTime
+        }
```

Comment:
Writing this explicitly because even if we would've written like this
```
this.player.currentTime = Math.max((this.previousPlayerTime - stepBackTime), 0)
```
Then the issue would be that it would explicitly set `currentTime` to `0` and `plyr` understands that as seeking, so when the video starts, the ugly recommendations are shown. 

If we do it like I've written, it will not set `currentTime` to `0` explicitly and it will start normally from 0.

Reply:
Nice 👍🏾